app/static/js/createEvents.js

```diff
@@ -484,3 +489,68 @@ $(".startDatePicker, .endDatePicker").change(function () {
 
   setCharacterLimit($("#inputCharacters"), "#remainingCharacters"); 
 });
+
+const selectedCohorts = [];
+$("input[name='cohorts[]']:checked").each(function() {
+  selectedCohorts.push($(this).val());
+});
+
+inviteCohorts(eventId, selectedCohorts)
+  .then(() => {
+    $("input[name='cohorts[]']:checked").prop('checked', false);
+  })
+  .catch(error => {
+    msgFlash("Error inviting cohorts: " + error.message, "danger");
+  });
+ 
+function inviteCohorts(eventId, selectedCohorts) {
```

Comment:
If we make the necessary changes in the routes then we don't need any of the javascript logic in this function since the data is already properly being passed in to the createEvent route.

Reply:
Done